app/views/admin/versions/_object_changes.html.haml

```diff
@@ -9,6 +9,8 @@
           %th Previous Value
         - if version.event != 'destroy'
           %th New Value
+        - if version.event == 'update' && !version.item.nil?
```

Comment:
@differentreality I want to move this check to ability.But this check should be invoked only when reverting a single attribute change.So this cannot be added to the ability with the current setup.